<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>573</width>
    <height>526</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>CyberSKA FITS Viewer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="CanvasWidget" name="canvasWidget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>481</width>
      <height>331</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>1</verstretch>
     </sizepolicy>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>481</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="coordinatesLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>281</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>8</pointsize>
          </font>
         </property>
         <property name="text">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Arial'; font-size:8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;x&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;y&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;z&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;stokes&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="valueLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>2</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>201</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>8</pointsize>
          </font>
         </property>
         <property name="text">
          <string>value</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="frameLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>2</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>201</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>8</pointsize>
          </font>
         </property>
         <property name="text">
          <string>frame</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QSlider" name="zoomSlider">
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>573</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpenFile"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
    <addaction name="separator"/>
    <addaction name="actionDBG"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="actionAbout"/>
    <addaction name="separator"/>
    <addaction name="actionChangelog"/>
    <addaction name="separator"/>
    <addaction name="actionAboutQt"/>
   </widget>
   <widget class="QMenu" name="menuCoordinates">
    <property name="title">
     <string>Coordinates</string>
    </property>
    <addaction name="actionCoordinatesB1950"/>
    <addaction name="actionCoordinatesJ2000"/>
    <addaction name="actionCoordinatesGalactic"/>
    <addaction name="actionCoordinatesEcliptic"/>
    <addaction name="separator"/>
    <addaction name="actionSexagecimal"/>
    <addaction name="actionDegrees"/>
    <addaction name="separator"/>
    <addaction name="actionGridOptions"/>
   </widget>
   <widget class="QMenu" name="menuMisc">
    <property name="title">
     <string>Misc</string>
    </property>
    <widget class="QMenu" name="menuPixel_Grid_Threshold">
     <property name="title">
      <string>Pixel Grid Threshold</string>
     </property>
     <addaction name="pixelGrid5Action"/>
     <addaction name="pixelGrid10Action"/>
     <addaction name="pixelGrid20Action"/>
     <addaction name="pixelGrid25Action"/>
     <addaction name="pixelGridOffAction"/>
    </widget>
    <addaction name="actionSaveScreenshot"/>
    <addaction name="actionFitsHeader"/>
    <addaction name="separator"/>
    <addaction name="menuPixel_Grid_Threshold"/>
    <addaction name="actionToggleToolbar"/>
   </widget>
   <widget class="QMenu" name="menuRegion_tools">
    <property name="title">
     <string>Region tools</string>
    </property>
    <addaction name="actionRegionStatistics"/>
    <addaction name="actionGaussianFitting"/>
   </widget>
   <widget class="QMenu" name="menuMovie">
    <property name="title">
     <string>Movie</string>
    </property>
    <addaction name="actionMoviePlayer"/>
    <addaction name="separator"/>
    <addaction name="actionPrecaching"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Color Tools</string>
    </property>
    <addaction name="actionHistogram"/>
    <addaction name="actionColormap"/>
   </widget>
   <widget class="QMenu" name="menuProfiles">
    <property name="title">
     <string>Profiles</string>
    </property>
    <addaction name="actionProfileX"/>
    <addaction name="actionProfileY"/>
    <addaction name="actionProfileZ"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuCoordinates"/>
   <addaction name="menuTools"/>
   <addaction name="menuRegion_tools"/>
   <addaction name="menuMovie"/>
   <addaction name="menuProfiles"/>
   <addaction name="menuMisc"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>Main Toolbar</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolTip { color: #006; font-size: 10pt; }</string>
   </property>
   <property name="allowedAreas">
    <set>Qt::AllToolBarAreas</set>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>30</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionMoviePlayer"/>
   <addaction name="actionHistogram"/>
   <addaction name="actionColormap"/>
   <addaction name="actionFitsHeader"/>
   <addaction name="actionProfileX"/>
   <addaction name="actionProfileY"/>
   <addaction name="actionProfileZ"/>
   <addaction name="actionRegionStatistics"/>
   <addaction name="actionGaussianFitting"/>
   <addaction name="actionGridOptions"/>
   <addaction name="actionSaveScreenshot"/>
   <addaction name="actionDBG"/>
  </widget>
  <action name="actionOpenFile">
   <property name="text">
    <string>&amp;Open...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="shortcut">
    <string>F4</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Icons/About.png</normaloff>:/Icons/About.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionChangelog">
   <property name="text">
    <string>Changelog</string>
   </property>
  </action>
  <action name="actionHistogram">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Icons/histogram64.png</normaloff>:/Icons/histogram64.png</iconset>
   </property>
   <property name="text">
    <string>Histogram...</string>
   </property>
   <property name="toolTip">
    <string>Histogram Options</string>
   </property>
  </action>
  <action name="actionColormap">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Icons/colormap64.png</normaloff>:/Icons/colormap64.png</iconset>
   </property>
   <property name="text">
    <string>Colormap...</string>
   </property>
   <property name="toolTip">
    <string>Colormap Options</string>
   </property>
  </action>
  <action name="actionCoordinatesB1950">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Equatorial (B1950)</string>
   </property>
  </action>
  <action name="actionCoordinatesJ2000">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Equatorial (J2000)</string>
   </property>
  </action>
  <action name="actionCoordinatesGalactic">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Galactic</string>
   </property>
  </action>
  <action name="actionCoordinatesEcliptic">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Ecliptic</string>
   </property>
  </action>
  <action name="actionDegrees">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Degrees</string>
   </property>
  </action>
  <action name="actionSexagecimal">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Sexagecimal</string>
   </property>
  </action>
  <action name="actionSaveScreenshot">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Icons/camera64.png</normaloff>:/Icons/camera64.png</iconset>
   </property>
   <property name="text">
    <string>Save Screenshot...</string>
   </property>
   <property name="iconText">
    <string>Screenshot</string>
   </property>
  </action>
  <action name="actionFitsHeader">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Icons/fitsHeader64.png</normaloff>:/Icons/fitsHeader64.png</iconset>
   </property>
   <property name="text">
    <string>FITS Header...</string>
   </property>
   <property name="iconText">
    <string>Fits
Header</string>
   </property>
   <property name="toolTip">
    <string>Show FITS Header</string>
   </property>
  </action>
  <action name="pixelGrid5Action">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </action>
  <action name="pixelGrid10Action">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>10</string>
   </property>
  </action>
  <action name="pixelGrid20Action">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>20</string>
   </property>
  </action>
  <action name="pixelGrid25Action">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>25</string>
   </property>
  </action>
  <action name="pixelGridOffAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Off</string>
   </property>
  </action>
  <action name="actionRegionStatistics">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Icons/sum.png</normaloff>:/Icons/sum.png</iconset>
   </property>
   <property name="text">
    <string>Region Statistics...</string>
   </property>
   <property name="iconText">
    <string>Region
Stats</string>
   </property>
  </action>
  <action name="actionGaussianFitting">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Icons/gaussian2d64.png</normaloff>:/Icons/gaussian2d64.png</iconset>
   </property>
   <property name="text">
    <string>Gaussian Fitting...</string>
   </property>
   <property name="iconText">
    <string>Gaussian
Fitting</string>
   </property>
   <property name="toolTip">
    <string>2D Gaussian Fitting</string>
   </property>
  </action>
  <action name="actionGridOptions">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Icons/grid64.png</normaloff>:/Icons/grid64.png</iconset>
   </property>
   <property name="text">
    <string>Grid Options...</string>
   </property>
   <property name="iconText">
    <string>Grid
Options</string>
   </property>
  </action>
  <action name="actionProfileX">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>X Axis...</string>
   </property>
   <property name="iconText">
    <string>X
Profile</string>
   </property>
   <property name="toolTip">
    <string>Profile along X axis</string>
   </property>
  </action>
  <action name="actionProfileY">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Y Axis...</string>
   </property>
   <property name="iconText">
    <string>Y
Profile</string>
   </property>
   <property name="toolTip">
    <string>Profile along Y axis</string>
   </property>
  </action>
  <action name="actionProfileZ">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Z Axis...</string>
   </property>
   <property name="iconText">
    <string>Z
Profile</string>
   </property>
   <property name="toolTip">
    <string>Profile along Z axis</string>
   </property>
  </action>
  <action name="actionMoviePlayer">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Icons/movie64.png</normaloff>:/Icons/movie64.png</iconset>
   </property>
   <property name="text">
    <string>Player...</string>
   </property>
   <property name="toolTip">
    <string>Movie Player</string>
   </property>
  </action>
  <action name="actionPrecaching">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Icons/cache64.png</normaloff>:/Icons/cache64.png</iconset>
   </property>
   <property name="text">
    <string>Allow Pre-Caching</string>
   </property>
   <property name="iconText">
    <string>Caching</string>
   </property>
  </action>
  <action name="actionToggleToolbar">
   <property name="text">
    <string>Toggle Toolbar</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionDBG">
   <property name="text">
    <string>DBG</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CanvasWidget</class>
   <extends>QWidget</extends>
   <header>CanvasWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
